SearchInfiniteSortedArray

```diff
@@ -0,0 +1,71 @@
+// Time Complexity  - O(log n)
+// Space Complexity - 0(1) - Constant space
+// Did this code successfully run on Leetcode :
+// Any problem you faced while coding this : Input - {3, 5, 7, 9, 10, 90, 100, 130, 140, 160, 170}
+// Yes, when I passed 160 and 170, I am getting ArrayIndexOutofBoundsException
+
```

Comment:
Yes that would happen. As the example used is a finite array. However in practice it will be an infinite array and we won't go out of bound ever